frontstage/views/session.py

```diff
@@ -1,14 +1,31 @@
-from datetime import datetime, timezone
-
-from flask import Blueprint, jsonify, request
+from flask import Blueprint, jsonify, make_response, redirect, request, session, url_for
 
 from frontstage.common.authorisation import jwt_authorization
+from frontstage.common.session import Session
 
 session_bp = Blueprint("session", __name__)
 
 
-@session_bp.route("/expires-at", methods=["GET"])
+@session_bp.route("/expires-at", methods=["GET", "PATCH"])
 @jwt_authorization(request, refresh_session=False)
-def session_expires_at(session):
-    expires_at = session.get_expires_in()
-    return jsonify(expires_at=datetime.fromtimestamp(expires_at, tz=timezone.utc).isoformat())
+def session_expires_at(expires_at):
+    refresh = None
+    if request.method == "PATCH":
+        refresh = True
+    return jsonify(expires_at=expires_at.get_formatted_expires_in(refresh))
+
+
+@session_bp.route("/session-expired", methods=["GET"])
+def session_expired():
```

Comment:
Isn't this the same as a logout and as such should be delegated? Unless I missed it there should also be a test

Reply:
As above

Reply:
This has been removed and the logout endpoint is now used.